.vale/styles/AsciiDoc/LinkContainsLinkText.yml

```
@@ -0,0 +1,9 @@
+---
+extends: existence
+scope: raw
+level: warning
+ignorecase: true
```

```suggestion
ignorecase: true
nonword: true
```

Think we should also set `nonword: true`. hmm.

---

@rohennes ?

---

I think so. Otherwise the angle brackets can be excluded from regex

https://github.com/errata-ai/vale/issues/643#issuecomment-1585131513